src/main/java/io/jenkins/plugins/gitlabserverconfig/servers/GitLabServer.java

```diff
@@ -243,30 +241,15 @@ public PersonalAccessToken getCredentials(AccessControlled context) {
         Jenkins jenkins = Jenkins.get();
         if (context == null) {
             jenkins.checkPermission(CredentialsProvider.USE_OWN);
-            return StringUtils.isBlank(credentialsId) ? null : CredentialsMatchers.firstOrNull( lookupCredentials(
+        } else {
+            context.checkPermission(CredentialsProvider.USE_OWN);
```

Comment:
this line is exactly like line 243.
I don't see the point for the "if" statement anymore... either way the code is the same (lines 243 and 245).
am I missing something?

Reply:
one is `context` the other is `jenkins`